yoti-sdk-api/src/test/java/com/yoti/api/client/docs/session/create/check/CheckType.java

```diff
@@ -0,0 +1,13 @@
+package com.yoti.api.client.docs.session.create.check;
+
+public enum CheckType {
```

Comment:
Think this is unused now that the extra test file ..api/client/docs/session/create/check/RequestedThirdPartyIdentityFraudOneCheckTest.java has been removed with second commit